src/shared/shared-types.d.ts

```diff
@@ -52,6 +52,7 @@ export interface DesktopSettingsType {
   activeTheme: string
   minimizeToTray: boolean
   syncAllAccounts: boolean
+  /** @deprecated is now not persisted between settings anymore */
```

Comment:
Will this variable be removed in the future?

Maybe this doc-string can be improved, how about "The last used file location for the save dialog is now only kept in memory and not persisted anymore."

On top, if you're planning to remove it, maybe also state why.